Core/src/OffConverter/namespace.h

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright (c) 2020 Technical University of Munich
+Chair of Computational Modeling and Simulation.
+
+TUM Open Infra Platform is free software; you can redistribute it and/or modify
+it under the terms of the GNU General Public License Version 3
+as published by the Free Software Foundation.
+
+TUM Open Infra Platform is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+#ifndef OpenInfraPlatform_Core_namespace_6e3a742e_afd1_4dd0_82a8_94ac0e7506b2_h
+#define OpenInfraPlatform_Core_namespace_6e3a742e_afd1_4dd0_82a8_94ac0e7506b2_h
+#if !defined OIP_NAMESPACE_OPENINFRAPLATFORM_CORE_OFFCONVERTER_BEGIN
+#define OIP_NAMESPACE_OPENINFRAPLATFORM_CORE_OFFCONVERTER_BEGIN \
+namespace OpenInfraPlatform { \
+namespace Core { \
+namespace OffConverter {
+#endif
+#if !defined OIP_NAMESPACE_OPENINFRAPLATFORM_CORE_OFFCONVERTER_END
+#define OIP_NAMESPACE_OPENINFRAPLATFORM_CORE_OFFCONVERTER_END \
+}\
+}\
+}
+#endif
+#if !defined EMBED_CORE_OFFCONVERTER_INTO_OIP_NAMESPACE
+#define EMBED_CORE_OFFCONVERTER_INTO_OIP_NAMESPACE(X) \
+namespace oip { \
+using OpenInfraPlatform::Core::OffConverter::X; \
+}
+#endif
+#endif // end define OpenInfraPlatform_Core_namespace_6e3a742e_afd1_4dd0_82a8_94ac0e7506b2_h
```

Comment:
Have you used the script provided in `Tools` to generate this file? If not, please do and update the file.

Reply:
Resolved: d9015af